hw/top_earlgrey/data/xbar_peri.hjson

```diff
@@ -41,9 +41,15 @@
       clock:     "clk_peri_i",
       reset:     "rst_peri_ni",
       pipeline: "false"
+    },
```

Comment:
@weicaiyang

Reply:
xbar TB is automatically updated for this change. LGTM